qcodes/instrument_drivers/basel/sp983c_remote.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any, Dict, Optional
+import numpy as np
+
+from qcodes.instrument.visa import VisaInstrument
+from qcodes.instrument.parameter import Parameter, DelegateParameter
+from qcodes.utils import validators as vals
+
+
+class SP983A(VisaInstrument):
+    """
+    A driver for Basel Preamp's (SP983C) Remote Instrument - Model SP983A.
+
+    Args:
+        name
+        address
+        input_offset_voltage: (Optional) A source input offset voltage
+            parameter. The range for input is -10 to 10 Volts and it is
+            user's responsibility to ensure this. This source parameter is
+            used to set offset voltage parameter of the preamp and the
+            source parameter should represent a voltage source that is
+            connected to the "Offset Input Volgate" connector of the SP983C.
+    """
+    def __init__(self,
+                 name: str,
+                 address: str,
+                 input_offset_voltage: Optional[Parameter] = None,
+                 **kwargs: Any) -> None:
+        super().__init__(name, address, terminator="\r\n", **kwargs)
```

Comment:
please set the terminator's default value in the `def init`, as an argument with the  default value, because this way the terminator can't be changed (if needed) without changing the driver code itself, an i'm not sure that we want that. i think we want to provide the corect default, but we also should give the users he flexibility to use a different terminator if they have to for an unknown reason

Reply:
updated as suggested